.github/workflows/stepci-2.8.2.yml

```
@@ -0,0 +1,27 @@
+name: stepci/stepci@2.8.2
+on:
+  pull_request:
+    paths:
+      - .github/workflows/reusable-docker-workflow.yml
+      - .github/workflows/stepci-2.8.2.yml
+  push:
+    branches:
+      - '*'
+    paths:
+      - .github/workflows/reusable-docker-workflow.yml
+      - .github/workflows/stepci-2.8.2.yml
+  schedule:
+    - cron: "16 1 * * *"
+  workflow_dispatch: ~
+
+permissions: read-all
+
+jobs:
+  stepci:
+    name: stepci
```

To align with other workflows, use the version here

```suggestion
  2.8.2:
    name: 2.8.2
```

My reasoning for this is that one workflow could test multiple versions of the same action (but I realize the version number in the workflow definition is kinda at odds with this idea 😅).